interp/src/debugger/commands/core.rs

```diff
@@ -299,6 +299,33 @@ impl From<(Vec<Path>, Option<PrintCode>, PrintMode)> for PrintTuple {
     }
 }
 
+#[derive(Debug, PartialEq, Clone)]
+pub enum ParseNodes {
+    Body,
+    Offset(u32),
+    If(bool),
+}
+pub struct ParsePath {
+    nodes: Vec<ParseNodes>,
+}
+
+impl ParsePath {
+    pub fn new(nodes: Vec<ParseNodes>) -> ParsePath {
+        ParsePath { nodes }
+    }
+
+    pub fn get_path(&self) -> Vec<ParseNodes> {
+        self.nodes.clone()
+    }
```

Comment:
Recommend making this a standard accessor which returns a reference to the vec
or a slice and cloning externally when necessary